bin/overrides.sh.erb

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+requote() {
```

Comment:
I have no idea what's going on here, so I'll talk to you about this file in person 🐼